.npmrc

```diff
@@ -0,0 +1 @@
+link-workspace-packages=deep
```

Comment:
I need this so that `pnpm -r run build` can build packages first then build examples. Using `"@prosemirror-adapter/lit": "workspace:*"` in package.json can do this job too, but it won't allow stackblitz to open a sub directory like `examples/lit` and install dependencies.